sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2595,14 +2664,20 @@ object Radar {
     }
 
     @JvmStatic
-    private fun logConversion(
+    internal fun sendLogConversionRequest(
         name: String,
-        data: JSONObject?,
-        location: Location?,
-        events: Array<RadarEvent>?,
-        user: RadarUser?,
+        data: JSONObject? = null,
+        location: Location? = null,
+        events: Array<RadarEvent>? = null,
+        user: RadarUser? = null,
         callback: RadarLogConversionCallback
     ) {
+        if (!initialized) {
+            callback.onComplete(RadarStatus.ERROR_PUBLISHABLE_KEY)
+
+            return
+        }
+
         apiClient.sendEvent(
             name,
             data,
```

Comment:
is `user` and `events` necessary to send? They are optional values here: https://github.com/radarlabs/radar-sdk-android/blob/develop/sdk/src/main/java/io/radar/sdk/RadarApiClient.kt#L1118:L1119

I believe it may be useful for context to look into the conversion events `POST /v1/events` endpoint in the server to get an understanding of how that function operates under the hood

Reply:
bump